src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -0,0 +1,222 @@
+        'status' : status})
+
+    containers = containers_mgmt_client_factory(None).list(rs_vault.name, resource_group, filter_string)
+    return get_one_or_many(get_paged_list(containers))
+
+def show_policy(client, policy_name, vault):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    policy = policy_mgmt_client_factory(None).get(rs_vault.name, resource_group, policy_name)
+    return policy
+
+def list_policies(client, vault):
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+
+    policies = policies_mgmt_client_factory(None).list(rs_vault.name, resource_group)
+    return get_one_or_many(get_paged_list(policies))
+
+def enable_protection_for_vm(client, vm, vault, policy):
+    policy_client = policy_mgmt_client_factory(None)
+
+    vm = get_vm_from_json(vm_mgmt_client_factory(None), vm)
+    rs_vault = get_vault_from_json(client, vault)
+    resource_group = get_resource_group_from_id(rs_vault.id)
+    policy = get_policy_from_json(policy_client, policy)
+
+    vm_name = vm.name
+    vm_rg = get_resource_group_from_id(vm.id)
+    
+    protectable_item = get_protectable_item(rs_vault.name, resource_group, vm_name, vm_rg)
+
+    container_uri = get_container_uri_from_id(protectable_item.id)
+    item_uri = get_item_uri_from_id(protectable_item.id)
+    
+    vm_item_properties = AzureIaaSComputeVMProtectedItem(policy_id=policy.id, source_resource_id=protectable_item.properties.virtual_machine_id)
+    vm_item = ProtectedItemResource(properties=vm_item_properties)
+
+    resp = protected_items_mgmt_client_factory(None).create_or_update(rs_vault.name, resource_group, "Azure", container_uri, item_uri, vm_item)
+
+    return resp
+
+################# Private Methods
+def get_protectable_item(vault_name, vault_rg, vm_name, vm_rg):
+    filter_string = get_filter_string({
+        'backupManagementType' : 'AzureIaasVM'})
+    
+    protectable_items_paged = protectable_items_mgmt_client_factory(None).list(vault_name, vault_rg, filter_string)
+    protectable_items = get_paged_list(protectable_items_paged)
+    
+    for protectable_item in protectable_items:
+        item_vm_name = get_vm_name_from_vm_id(protectable_item.properties.virtual_machine_id)
+        item_vm_rg = get_resource_group_from_id(protectable_item.properties.virtual_machine_id)
+        if item_vm_name == vm_name and item_vm_rg == vm_rg:
+            return protectable_item
+    # we're still here, do discovery
+
+def get_paged_list(obj_list):
+    from msrest.paging import Paged
+
+    if isinstance(obj_list, Paged):
+        return list(obj_list)
+    else:
+        return obj_list
+
+def get_one_or_many(obj_list):
+    if len(obj_list) == 1:
+        return obj_list[0]
+    else:
+        return obj_list
+
+def get_item_type(workload_type):
+    if workload_type == "AzureVM":
+        return "VM"
```

Comment:
`return "VM" if workload_type == "AzureVM" else None`
This would be a lot clearer to indicate that if the result is False, you want to return None.
Right now, it's implicit and so less clear.